src/getHumanAge.test.js

```diff
@@ -7,4 +7,28 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0 for ages less than the first threshold', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return1for ages between the first and second thresholds', () => {
```

Comment:
There is a typo in the test description: 'should return1for ages between the first and second thresholds'. It should be 'should return 1 for ages between the first and second thresholds' with a space between 'return' and '1'.